src/compositemark/common.ts

```diff
@@ -21,7 +21,7 @@ export function partLayerMixins<P extends PartsMixins<any>>(
   const {color, opacity} = markDef;
 
   const mark = markDef.type;
-  const partMarkDef: MarkConfig = markDef[part];
+  const partMarkDef: MarkConfig = isBoolean(markDef[part]) ? {} : <MarkConfig>markDef[part];
```

Comment:
This is incorrect as `false` would not disable the part.

Reply:
Also, `...partMarkDef` below will apply `...true`, which is not correct too.

Reply:
isn't that this line
```if (markDef[part] || (markDef[part] === undefined && compositeMarkConfig[part])) {```
has already handled the case?

my `markDef[part]` in errorbar also has to be `boolean` or `MarkConFig`.

Reply:
Ah my bad.